subprojects/core-api/src/main/java/org/gradle/api/initialization/dsl/ScriptHandler.java

```diff
@@ -103,6 +105,25 @@
      */
     ConfigurationContainer getConfigurations();
 
+    /**
+     * Configures dependency locking
+     *
+     * @param configureClosure the configuration action
+     * @since 6.1
+     */
+    @Incubating
+    void dependencyLocking(Closure configureClosure);
```

Comment:
:o: Do you need a `Closure` here? I think for new things you can just use `Action` and let Gradle do the decoration, no?

Reply:
Looks like I do need a `Closure`. I first tried with an `Action` but then:
```
buildscript {
    dependencyLocking {
        lockMode = LockMode.LENIENT
    }
}
```
was invalid in a Groovy build script.

Reply:
I guess it means that hte script handler is not created via an instantiator.